Ontology/Willow/Capability/State/Binary State/Alarm State/Overload/CurrentOverloadAlarmState.json

```diff
@@ -0,0 +1,17 @@
+{
+  "@id": "dtmi:com:willowinc:CurrentOverloadAlarmState;1",
```

Comment:
Is this different than the other Current Alarm States such as HighCurrentAlarmState?

Reply:
Yes, the two models are borne of different devices. For the HighCurrentAlarmState model, a current sensor measures the current and another device compares that to an alarm threshold that's typically adjustable. CurrentOverloadAlarmState would come from a circuit breaker or similar device that has an alarm contact to indicate when it has tripped.